vendir.lock.yml

```diff
@@ -93,7 +93,7 @@ directories:
     path: node-feature-discovery
   - helmChart:
       appVersion: v1.0.0
-      version: 0.3.27
```

Comment:
It seems that the vendir file was updated, but is it possible you did not commit the vendir.yml file but committed the lock file?